src/services/BalanceUtilities.tsx

```diff
@@ -12,6 +12,7 @@ import { notify, notifySuccess } from './FeedbackService'
 
 const KILT_COIN = 1
 const KILT_MICRO_COIN = 1_000_000
+const KILT_FEMTO_COIN = '1000000000000000'
```

Comment:
We might want to discuss changing the scaling, as this gives us 1152 kilt Coins on the faucet.

Previously we had 1 unit = 1 micro kilt, now we have 1 unit  = 1 femto coin, and in reverse we had 1000000 micro kilt = 1 kilt coin, now we have 1000000000000000 femto klt = 1 kilt coins, which makes our balance 10⁻⁹ times smaller.

Theres also some issues with the way balance changes are reported, the way we transfer coins from one account to the other, and so on.

We should settle for an appropriate scaling.

Reply:
solved with https://github.com/KILTprotocol/mashnet-node/pull/91